README.md

```diff
@@ -1 +1,72 @@
 # ironfoudation-team-project
+# WOOF application 
+
+## Table of contents
+* [General info](#general-info)
+* [Technologies](#technologies)
+* [Installation](#installation)
+* [Sources](#sources)
+* [Authors](#authors)
+* [Important](#important)
+* [Notes](#notes)
+
+## General info
+
+This is the application for iOS.
+Woof application connects dog owners with pet sitters which contributes to build trusted relationships between them. The app makes it easy for nearby pet owners to find pet sitting and makes convienience of responding to jobs for pet sitters from phone.
+App also makes managing sitting bussines and pet owner's schedule. They can choose their own schedule, prices, and requirements for servise. 
+
+**Features include**
+-Secure store all of pet owners' and pet sitters' data base. 
+-Regularly data base update with personal information.
+
+**It also important to know:**
+To process application,some private cliets information like home adress, mobile phone is needed.
+
+## Technologies
+
+Project is created with:
+* macOS ventura 13.3
+* Xcode version: 14.3
+* Swift: 5.8
+* SDKs for IOS: 16.4
+* SwiftLint: 0.51.0
+* SwiftFormat: 0.51.7
+
+## Installation
+
+To run this project:
+```
+1)install it locally by cloning the current repository, using: git@github.com:ios-course/ironfoudation-team-project.git
+2)run the file Woof.xcodeproj
+```
+## Sources
+
+This app uses the folowing API and libraries:
+//*This section will be updated during the application development*  
+
+## Authors
+
+You can contact the authors by the following email adresses:
+```
+Nadia Shpakouskaya: shpakovna@gmail.com 
+Alex Zavadskiy: fh.zavadskii@gmail.com
+Julya Kmita: julyakmita@gmail.com
+```
+
+## Important:
+
+### Creating a new branch:
+1) all brunches are created from the main branch;
+2) name of a new branch should show the issue type, issue number and branch name:
+ ```
+ [type_of_task - #issue_from_backlog - title].
+ For example:
+ feature-1-create-project.
+ ```
+### Merge restrictions to the main branch:
+-minimum two approvals;
+-all conversations should be closed (resolve) after exhaustive discussion.
```

Comment:
Maybe better move this information to board_rules or another place.
I don't think, that this information is important for users.

Reply:
I agree. The information is removed to the board_rules file.